incuna_auth/middleware/utils.py

```diff
@@ -0,0 +1,16 @@
+import re
+
+# Python 2/3 compatibility hackery
+try:
+    unicode
+except NameError:
+    unicode = str
```

Comment:
I found https://docs.djangoproject.com/en/1.7/topics/python3/#str-and-unicode-methods, but I'm a bit confused as to how I apply that to a string literal.  Halp?